streampark-console/streampark-console-service/src/test/java/org/apache/streampark/console/core/service/application/ApplicationActionServiceTest.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.console.core.service.application;
+
+import org.apache.streampark.console.SpringUnitTestBase;
+import org.apache.streampark.console.core.entity.Application;
+
+import com.baomidou.mybatisplus.core.conditions.query.QueryWrapper;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/** org.apache.streampark.console.core.service.alert.ApplicationActionServiceTest. */
+class ApplicationActionServiceTest extends SpringUnitTestBase {
+  @Autowired private ApplicationActionService applicationActionService;
+
+  @AfterEach
+  void cleanTestRecordsInDatabase() {
+    applicationActionService.remove(new QueryWrapper<>());
+  }
+
+  @Test
+  void testUpdateApplicationActionServiceById() {
+    Application app = new Application();
```

Comment:
I'm not sure what the significance of this test is. In my opinion, it's just an interface similar to updating dto. Do we need to write a test class specifically for this?